Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM alpine as make_dist
+
+RUN apk update && apk add \
+    make \
+    bash \
+    tar \
+    python3
+
+# If build_src is alredy present, shouldn't need to redownload
+COPY build_src build_src
+COPY Makefile .
+
+RUN make clean && make dist/kolibri_archive.tar.gz build_src/VERSION
+
+
+FROM ubuntu:bionic
```

Comment:
I am yet to test this, but @jredrejo do you recall if building on non-Trusty was the reason why the .deb didn't work on Trusty?

(building the .deb on/for later versions of Ubuntu means that it won't work on Trusty)

This is not to say that we need to spend a lot of time getting this to work, but if it's desirable, it would be nice to know if it could work... gonna check back!

Reply:
Tested this - 14.04 needs to be upgraded, and it can run the version built on Buildkite, also did an extra local build of 0.13.2 on Bionic and it worked, too.

The failure to run on 14.04 is reproducible before running `apt upgrade`, I'm not sure about which version of dpkg that fixed it, but it doesn't matter. I'll file a docs issue about 14.04 support.